### PR TITLE
fix: bug when creating CE from LR

### PR DESCRIPTION
**Pull request by gurneyalex**
if no consignee was set on the LR (possible if LR is Cost Estimate only)
then creating the CE would crash because we were wrongly using the consignee as
partner when playing the sale.order onchanges

### CODE REVIEW

File: logistic_requisition/test/requisition_create_cost_estimate_only.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        location_obj = self.pool['stock.location']
+        source = self.browse(cr, uid, source_line_id)
+        sale_lines = source.requisition_id.line_ids[0].cost_estimate_id.order_line
+        sale_line = [line for line in sale_lines if line.product_id == source.proposed_product_id][0]
+        line = source.requisition_line_id
+        assert sale_line.product_id == source.proposed_product_id, (
+            "product_id should be %s, received %s" %
+            (source.proposed_product_id, sale_line.product_id))
+
+        assert sale_line.name == line.description, (
```
Comment (lepistone) [inline logistic_requisition/test/requisition_create_cost_estimate_only.yml:92]: I know that you know, but are you sure you don't want to import `nose.tools` assertions? I doubt it gets more little :cat: killed. (not blocking over that)

**Comment by lepistone** [comment]
Code review OK with a remark on assertions. Once green, :+1:

**Comment by lepistone** [comment]
I removed a `print` statement found by pylint.

**Comment by lepistone** [comment]
There is a new test failure related to something I worked on on another PR. I am investigating that.

**Comment by nbessi** [comment]
:+1:  on principe but I'surprised that change do not impact more code especially the one run when
confirming the CE

**Comment by lepistone** [comment]
I'm not entirely happy about long stories in the tests, but it's good to have those anyway, and we'll be able  to improve them later.

Thanks!

**Pull request merged**